examples/soul2vr.py

```diff
@@ -0,0 +1,142 @@
+from email.base64mime import body_decode
```

Comment:
what's this?

Reply:
This sentence is unnecessary, it is my mistake.
I removed the sentence.